### ISSUE TITLE
it suddenly quits so many times on macOS Ventura

### ISSUE THREAD

**Issue by KyleDEV** [sqlitebrowser/sqlitebrowser#3274]
I have been using it with an sqlite db file that has a password
It quits suddenly whatever I do.

Useful extra information
-------------------------
> DB4S v3.12.2 [built for x86_64-little_endian-lp64] on macOS 10.16 (darwin/22.2.0) [x86_64]
> using SQLCipher Version 4.4.3 community (based on SQLite 3.34.1)
> and Qt 5.15.2

**Comment by lucydodo** [comment]
That's weird. When I just tested<sup>1</sup> it, there was no problem.
Can you post a crash dump or something to help me troubleshoot?

Also, could you please reset the app settings and test again before that (or on another mac)?
The settings file for DB4S is located in the following location:
`$HOME/Library/Preferences/net.sourceforge.sqlitebrowser.plist`

> 1: Encrypted DB by SQLCipher 4 defaults
with DB4S 3.12.2, Qt 5.15.2, SQLCipher 4.4.3, SQLite 3.34.1
on macOS Ventura 13.2 (22D49)

**Comment by justinclift** [comment]
@KyleDEV Which model of Mac are you using?

I'm kind of wondering if you're using an M1 or M2 based Mac, in which case recently nightly builds for that would probably be more stable. :smile:

**Comment by KyleDEV** [comment]
How cna I reset? I deleted the plist and opened the db again. and it has made one again. is it the right way to reset?
I have an Intel Mac not m1,m2 thing. 
My DB file is on an external drive with .db extention. the table is from .csv file with only 50 records.

**Comment by lucydodo** [comment]
You're right about the reset method. Why don't you give the nightly build a try then?
It's been a while since we released a stable version, so the nightly build might be more stable.

As of today, here are the links to the latest nightly builds for Intel Macs.
https://nightlies.sqlitebrowser.org/latest/DB.Browser.for.SQLite-sqlcipher-intel.dmg

**Comment by justinclift** [comment]
@KyleDEV Did you have a chance to try out the nightly build?

**Comment by KyleDEV** [comment]
@justinclift I am trying. I need some time to use it for now i am doing other thins. thanks
